### ISSUE TITLE
`import/no-empty-named-blocks` false positive in many cases

### ISSUE THREAD

**Issue by zloirock** [import-js/eslint-plugin-import#2667]
![image](https://user-images.githubusercontent.com/2213682/211978254-e727ff6f-afb1-4f51-93f2-ca5aa63313c1.png)

Triggers from the last 3 files:
```js
import { DESCRIPTORS, NODE } from '../helpers/constants';
// ...
import { timeLimitedPromise } from '../helpers/helpers';
// ...
import { DESCRIPTORS } from '../helpers/constants';
```
In every file all errors on 1:1 position. No one of them has empty named blocks.

**Comment by ljharb** [comment]
hmm, i can't seem to reproduce these. What eslint parser are you using? (i tried the default one)

**Comment by ljharb** [comment]
Either way, the entire implementation of this rule needs to be rewritten to solve #2666, so hopefully the next release will address this issue.

**Comment by ljharb** [comment]
v2.27.3 is released; give that a shot.

**Comment by zloirock** [comment]
It seems it's fixed in 2.27.4.
